server/service/src/invoice/inbound_shipment/generate.rs

```diff
@@ -24,7 +24,7 @@ pub fn generate_empty_invoice_lines(
                     location_id: None,
                     batch: None,
                     expiry_date: None,
-                    pack_size: 1.0,
+                    pack_size: item.default_pack_size,
```

Comment:
Per the store pref `pack_to_one` thing - I think we actually need to factor in the same thing here!

You can use the `get_store_preferences` function, like here:
https://github.com/msupply-foundation/open-msupply/blob/20f5074f9cd298f89ad180cf04ea110d629da2b5/server/service/src/invoice_line/stock_in_line/insert/generate.rs#L30

If pack_to_one is true, should be 1.0, otherwise can use the default pack size 🙏